implementations/rust/ockam/ockam_vault/Cargo.toml

```diff
@@ -30,14 +30,13 @@ disable_default_noise_protocol = []
 OCKAM_XX_25519_AES256_GCM_SHA256 = []
 OCKAM_XX_25519_AES128_GCM_SHA256 = []
 OCKAM_XX_25519_ChaChaPolyBLAKE2s = []
+aws-lc = ["dep:aws-lc-rs"]
```

Comment:
What are the downside for using aws lib? If it's better, should we use it by default?

Reply:
It requires `cmake` and `libclang`, also it doesn't support `no_std`